<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPUv3Class</class>
 <widget class="QMainWindow" name="GPUv3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPUv3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show specification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add graphics card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add gaming graphics card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add sound card</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GPUv3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>GPUv3Class</receiver>
   <slot>deleteCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>GPUv3Class</receiver>
   <slot>specification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>GPUv3Class</receiver>
   <slot>addGamingGraphicsCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>GPUv3Class</receiver>
   <slot>addSoundCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>GPUv3Class</receiver>
   <slot>addGraphicsCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addGraphicsCard()</slot>
  <slot>deleteCard()</slot>
  <slot>specification()</slot>
  <slot>addGamingGraphicsCard()</slot>
  <slot>addSoundCard()</slot>
 </slots>
</ui>
